src/Umbraco.Infrastructure/Manifest/ManifestParser.cs

```diff
@@ -217,6 +224,31 @@ public PackageManifest ParseManifest(string text)
         return manifest;
     }
 
+    private string? GetAssemblyVersion(string name)
+    {
+        foreach (Assembly assembly in AssemblyLoadContext.All.SelectMany(x => x.Assemblies))
```

Comment:
I initially used `ITypeFinder.AssembliesToScan` (to have a cached list of assemblies), but that only includes assemblies that have references to Umbraco (since that's what `IAssemblyProvider.Assemblies` returns in the `DefaultUmbracoAssemblyProvider` implementation). This contains the following comment:

https://github.com/umbraco/Umbraco-CMS/blob/60a5b19dc9fe24f5379a253e1f3fdcb9cb537dfc/src/Umbraco.Core/Composing/DefaultUmbracoAssemblyProvider.cs#L32-L37

So to have access to all assemblies, I've used `AssemblyLoadContext.All` for now, but since package assemblies are loaded on start-up (and not dynamically generated/loaded), it might be enough to change this to:

```suggestion
        foreach (Assembly assembly in AssemblyLoadContext.Default.Assemblies)
```

Reply:
I can confirm `AssemblyLoadContext.Default` will have all the package assemblies 👍🏻